package.json

```diff
@@ -5,11 +5,16 @@
   "description": "A class designed to enable easily extending the built-in javascript Error, allowing typed errors.",
   "main": "lib/typed-error.js",
   "types": "lib/typed-error.d.ts",
+  "engines": {
+    "node": ">=12.0.0",
```

Comment:
This change will block clients such as the CLI from updating. I don’t expect we’ll need to do many changes to typed-error that clients will need, but it will take many years until they’ll be able to require Node 12 so I’m not sure this change makes sense. What is the motivation of this in the first place?

Reply:
@dfunckt yeah I was wondering if I had been a bit keen bumping this module too when I later looked at the dependants. I'll remove the breaking changes as they don't provide huge benefit to this module anyway.